import-service/src/functions/importFileParser/handler.ts

```diff
@@ -0,0 +1,108 @@
+import type { ValidatedEventAPIGatewayProxyEvent } from '@libs/api-gateway';
+import { middyfy } from '@libs/lambda';
+import schema from './schema';
+import AWS from 'aws-sdk';
+const s3 = new AWS.S3({ region: 'us-east-1' });
+const BUCKET = 'node-js-aws-s3-task5'
+import csv from 'csv-parser';
+
+const importFileParser: ValidatedEventAPIGatewayProxyEvent<typeof schema> = async (event) => {
+    try {
+        console.log(`Lambda invocation with: ${JSON.stringify(event)}`);
+        const s3Instance = s3;
+const bucketName = BUCKET
+const uploadFolderName = 'uploaded'
+const parsedForlderName = 'parsed'
+const sqs = new AWS.SQS();
```

Comment:
Better create AWS Service Clients outside the Lambda. That way to reduce execution time by sharing the client among them. Check AWS Lambda best practices

Reply:
Thanks @IvanRadchenko, Updated.

https://github.com/shekharverma17/shop-backend-services-aws/commit/e4667cdc9c0475871f14a192dc1d605fd2d0b072